src/wp-includes/class-wp-theme-json.php

```diff
@@ -1764,12 +1764,17 @@ protected static function compute_preset_classes( $settings, $selector, $origins
 	 * </code>
 	 *
 	 * @since 5.9.0
+	 * @since 6.6.0 Added early return if missing scope or selector.
 	 *
 	 * @param string $scope    Selector to scope to.
 	 * @param string $selector Original selector.
 	 * @return string Scoped selector.
 	 */
 	public static function scope_selector( $scope, $selector ) {
```

Comment:
Would be nice if there were a test for this, but given that it's an existing function I think it's fine to do that separately (and tests can be added anytime during the release cycle).